### PR TITLE
ONEUP-637: Add new field

### PR DESCRIPTION
**Pull request by catalin-cretu**
- add new field tagsOperator

### CODE REVIEW

File: pressed/src/test/scala/de/welt/contentapi/pressed/models/ApiPressedContentTest.scala [hunk 1]
```diff
@@ -77,6 +77,10 @@ class ApiPressedContentTest extends PlaySpec {
       apiPressedContent.relatedByRole() mustBe empty
     }
 
+    "return content with empty tagsOperator" in {
```
Comment (aramin) [inline pressed/src/test/scala/de/welt/contentapi/pressed/models/ApiPressedContentTest.scala:80]: some more context would be nice. what is a tags operator used for?

File: core/src/main/scala/de/welt/contentapi/core/models/ApiContent.scala [hunk 2]
```diff
@@ -75,6 +75,7 @@ case class ApiContent(webUrl: String,
                       var roles: Option[List[String]] = None,
                       sections: Option[ApiSectionData] = None,
                       tags: Option[List[ApiTag]] = None,
+                      tagsOperator: Option[String] = None,
```
Comment (aramin) [inline core/src/main/scala/de/welt/contentapi/core/models/ApiContent.scala:78]: please provide the scaladoc description for the new parameter

File: pressed/src/test/scala/de/welt/contentapi/pressed/models/ApiPressedContentTest.scala (hunk 1 above)
Comment (catalin-cretu) [reply pressed/src/test/scala/de/welt/contentapi/pressed/models/ApiPressedContentTest.scala:80]: Why do we have a changelog? Who else is interested in these changes besides our team?

**Comment by catalin-cretu** [comment]
> please bump the version in `build.sbt` and write a short description in `Changelog.md`

It looks like the version number is going to be derived based on the build number so we don't need to do anything.

Why do we have a changelog? Who else is interested in these changes besides our team?

File: core/src/main/scala/de/welt/contentapi/core/models/ApiContent.scala (hunk 2 above)
Comment (catalin-cretu) [reply core/src/main/scala/de/welt/contentapi/core/models/ApiContent.scala:80]: Done

**Comment by aramin** [comment]
> > please bump the version in `build.sbt` and write a short description in `Changelog.md`
> 
> It looks like the version number is going to be derived based on the build number so we don't need to do anything.
> 
> Why do we have a changelog? Who else is interested in these changes besides our team?

the changelog is only for us as some sort of documentation. i'm not a big fan of if but i mentioned it for consistency reasons. and yes, the patch version is derived on the build number but as this is a new field and not a bugfix i would bump the minor version

File: pressed/src/test/scala/de/welt/contentapi/pressed/models/ApiPressedContentTest.scala (hunk 1 above)
Comment (catalin-cretu) [reply pressed/src/test/scala/de/welt/contentapi/pressed/models/ApiPressedContentTest.scala:80]: I've added the docs for the param. Do those docs make sense?

Comment (aramin) [reply pressed/src/test/scala/de/welt/contentapi/pressed/models/ApiPressedContentTest.scala:80]: yes 👍

**Pull request merged**